plat/hisilicon/poplar/include/platform_def.h

```diff
@@ -96,6 +95,11 @@
 #endif /* SPD_none */
 #endif
 
+#define POPLAR_EMMC_DATA_BASE 0x02200000
```

Comment:
Static check - Add U / ULL suffix. Could use the U() ULL() macro from utils_def.h